### ISSUE TITLE
Error when I clicked in a sketch file

### ISSUE THREAD

**Issue by alvescleiton** [avocode/avocode#2138]
---
* **Version**: 3.4.1
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 192687
---
```
TypeError: Cannot read property '$$' of undefined
    at RegisteredPointer.nonConstNoSmartPtrRawPointerToWireType [as toWireType] (C:\Users\cleiton\AppData\Local\avocode\app-3.4.1\resources\app.asar\node_modules\@avocode\rendering-emscripten\rendering-asm.js:4:574344)
    at Object.destroyOutputAttributes (eval at new_ (C:\Users\cleiton\AppData\Local\avocode\app-3.4.1\resources\app.asar\node_modules\@avocode\rendering-emscripten\rendering-asm.js:1:1), <anonymous>:7:26)
    at e.destroyOutputAttributes (C:\Users\cleiton\AppData\Local\avocode\app-3.4.1\resources\app.asar\node_modules\@avocode\stage\dist\entities\rendering.js:1:5294)
    at e.finalize (C:\Users\cleiton\AppData\Local\avocode\app-3.4.1\resources\app.asar\node_modules\@avocode\stage\dist\entities\rendering-context\javascript-rendering-context.js:1:5483)
    at t.finalize (C:\Users\cleiton\AppData\Local\avocode\app-3.4.1\resources\app.asar\node_modules\@avocode\stage\dist\artboards\artboard.js:1:10627)
    at file:///C:/Users
```

**Comment by jakubzitny** [comment]
Hi @ariscratle, thanks for reporting this. It's a bug in our rendering engine and we'll try to fix it, we've seen it already in #1978. How often is it happening, please?

**Comment by jakubzitny** [comment]
Hi @ariscratle, thanks for reporting this. It's a bug in our rendering engine and we'll try to fix it, we've seen it already in #1978. How often is it happening, please?

**Comment by jakubzitny** [comment]
Closing due to inactivity, please let us know if you have any details.
